publication/src/main/scala/hmda/publication/reports/disclosure/DisclosureReportPublisher.scala

```diff
@@ -77,6 +75,8 @@ class DisclosureReportPublisher(supervisor: ActorRef) extends HmdaActor with Loa
     D71, D72, D73, D74, D75, D76, D77,
     D81, D82, D83, D84, D85, D86, D87,
     D11_1, D11_2, D11_3, D11_4, D11_5, D11_6, D11_7, D11_8, D11_9, D11_10,
+    A1, A2, A3,
+    A1W, A2W, A3W,
```

Comment:
The reports with `W` in the name are nationwide, so they should only be generated once per institution, rather than once per MSA. You don't have to make that change in this PR; I'm going to open a PR with report BW and make the change there. Just wanted to let you know.

Reply:
Yeah, definitely an good point there. The rationale behind generating it this way is that the FFIEC site displays this report in each MSA.  Given our current folder structure and the unknown UI aspect, along with the current publication process, I figured this would be the easiest way to have the report appear in all MSAs.

I'll check out BW when it comes along 👍

Reply:
Oh, interesting. I see what you mean. I'll consider that. 

I'll see if it's actually important to have those reports in every MSA, and if there's a way to do so without using the computation power to re-generate it several times.